setup.py

```diff
@@ -268,6 +268,7 @@ def readme():
         "scikit-learn>=0.20",
         "scipy",
         "pynndescent>=0.4",
+        "annoy",
```

Comment:
Please set this to be `annoy>=` whatever the version you have installed.

Reply:
scipy does not have `>=` in here, is it on purpose?

Reply:
scipy is required by a lot of packages and it's all too easy to cause requirement conflicts. Also, scipy is really good at back-compatibility.

Reply:
ok done